src/components/form/examples/help-form-schema.ts

```diff
@@ -0,0 +1,84 @@
+export const schema = {
+    type: 'object',
+    properties: {
+        address: {
+            type: 'object',
+            title: 'Book a demo',
+            properties: {
+                planet: {
+                    type: 'string',
+                    title: 'Name',
+                    description: 'Please write your name & surname',
+                    lime: {
+                        help: {
+                            value: '**Why do we need this information?** <br> This enables our sales representatives to contact you personally. We will keep all your personal information safe!',
+                            readMoreLink: {
+                                href: 'https://www.lime-technologies.com/en/av-saas/',
+                                title: 'This link opens in a new tab',
+                                target: '_blank',
+                                text: 'Our Terms & Privacy Policies',
+                            },
+                        },
+                    },
+                },
+                galaxy: {
+                    type: 'string',
+                    title: 'Company',
+                    description: 'Please write your company name',
+                },
+                email: {
+                    type: 'string',
+                    title: 'Work Email',
+                    format: 'email',
+                    lime: {
+                        help: {
+                            value: 'Enter a valid email address so that we can contact you. We will not share this information with any third parties.',
+                            readMoreLink: {
+                                href: 'https://www.lime-technologies.com/en/av-saas/',
+                                title: 'This link opens in a new tab',
+                                target: '_blank',
+                                text: 'Our Terms & Privacy Policies',
+                            },
+                        },
+                    },
+                },
+                race: {
```

Comment:
`race` and `galaxy`?

This was perhaps copied and pasted in a bit of a hurry…? 😝